CI: ERROR: (gcloud.container.clusters.create) ResponseError: code=400, message=No valid versions with the prefix "1.24.5" found.

```
Run gcloud container clusters create cilium-cilium-3944928056 \
  gcloud container clusters create cilium-cilium-39449[2](https://github.com/cilium/cilium/actions/runs/3944928056/jobs/6751353282#step:12:2)8056 \
    --labels "usage=cilium-cilium,owner=2[3](https://github.com/cilium/cilium/actions/runs/3944928056/jobs/6751353282#step:12:3)1[4](https://github.com/cilium/cilium/actions/runs/3944928056/jobs/6751353282#step:12:4)7" \
    --zone us-west2-a \
    --cluster-version 1.24.[5](https://github.com/cilium/cilium/actions/runs/3944928056/jobs/6751353282#step:12:5) \
    --enable-ip-alias \
    --create-subnetwork="" \
    --image-type COS_CONTAINERD \
    --num-nodes 2 \
    --machine-type e2-custom-2-409[6](https://github.com/cilium/cilium/actions/runs/3944928056/jobs/6751353282#step:12:6) \
    --disk-type pd-standard \
    --disk-size 10GB \
    --node-taints ignore-taint.cluster-autoscaler.kubernetes.io/cilium-agent-not-ready=true:NoExecute \
    --preemptible
  shell: /usr/bin/bash -e {0}
  env:
    clusterName: cilium-cilium-3944928056
    zone: us-west2-a
    k8s_version: 1.24.5
    cilium_cli_version: v0.12.12
    check_url: https://github.com/cilium/cilium/actions/runs/3944928056
Default change: During creation of nodepools or autoscaling configuration changes for cluster versions greater than 1.24.1-gke.[8](https://github.com/cilium/cilium/actions/runs/3944928056/jobs/6751353282#step:12:8)00 a default location policy is applied. For Spot and PVM it defaults to ANY, and for all other VM kinds a BALANCED policy is used. To change the default values use the `--location-policy` flag.
Note: The Pod address range limits the maximum size of the cluster. Please refer to https://cloud.google.com/kubernetes-engine/docs/how-to/flexible-pod-cidr to learn how to optimize IP address allocation.
ERROR: (gcloud.container.clusters.create) ResponseError: code=400, message=No valid versions with the prefix "1.[24](https://github.com/cilium/cilium/actions/runs/3944928056/jobs/6751353282#step:12:24).5" found.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/conformance-gke-v1.12.yaml
+++ b/.github/workflows/conformance-gke-v1.12.yaml
@@ -63,7 +63,7 @@ concurrency:
 env:
   clusterName: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}
   zone: us-west2-a
-  k8s_version: 1.24.5
+  k8s_version: 1.24
   cilium_cli_version: v0.12.12
   check_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
   USE_GKE_GCLOUD_AUTH_PLUGIN: True

--- a/.github/workflows/conformance-gke-v1.13.yaml
+++ b/.github/workflows/conformance-gke-v1.13.yaml
@@ -63,7 +63,7 @@ concurrency:
 env:
   clusterName: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}
   zone: us-west2-a
-  k8s_version: 1.24.5
+  k8s_version: 1.24
   cilium_cli_version: v0.12.12
   check_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
   USE_GKE_GCLOUD_AUTH_PLUGIN: True

--- a/.github/workflows/conformance-gke.yaml
+++ b/.github/workflows/conformance-gke.yaml
@@ -69,7 +69,7 @@ env:
   cilium_cli_version: v0.12.12
   check_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
   USE_GKE_GCLOUD_AUTH_PLUGIN: True
-  k8s_version: 1.24.5
+  k8s_version: 1.24
 
 jobs:
   check_changes:

